mantidproject/mantid
# EnggDiffractionGUI::Fixes crashes when run number given for Pre-processing Rebin

fixes #14886 

This branch should disable Mantid from crash when a valid or an invalid run number is given to `Interfaces/Engineering Diffraction/Pre-processing`.

**To Test**
- Go to `Interfaces/Engineering Diffraction/`
- Enter anything for RB Number `e.g: 13245` (used for files output)
- Go to `Pre-proceessing tab`
- Enter valid run number (make sure the directory is found in `Manage User Directories`) 
  
  > - error on invalid or empty run number
- Click `Rebin`
- One of the valid run number that could be tested is `197019`, you can find the file `ENGINX00197019` in the system test data. 

_For any initial setting or having any trouble setting up, please slack me._
